dev-tex/rubber/rubber-1.5.1.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_5 python3_6 python3_7 )
```

Comment:
After EAPI, there should be an empty line.

Reply:
Done, also had another look at array variant and decided for it.

Reply:
You could shorten this to `python3_{5,6,7}`